config/winston.js

```diff
@@ -1,16 +1,26 @@
-import winston from 'winston';
+import config from './config';
 
-const logger = new (winston.Logger)({
+const { createLogger, transports, format } = require('winston');
+
+const { printf, timestamp, combine, colorize } = format;
```

Comment:
If you want to be fancy, this also works:
```
const { createLogger, transports, format: { printf, timestamp, combine, colorize} } = require('winston');
```
Also, is there a reason you use `const ... require()` instead of `import ... from`?